examples/v1beta1/pipeline/realtimeimageclassfication/README.md

```diff
@@ -0,0 +1,142 @@
+```
+
+after successful install you are expected to see the running kafka cluster
+```bash
+NAME                      READY   STATUS    RESTARTS   AGE
+my-kafka-cp-kafka-0       2/2     Running   0          126m
+my-kafka-cp-kafka-1       2/2     Running   1          126m
+my-kafka-cp-kafka-2       2/2     Running   0          126m
+my-kafka-cp-zookeeper-0   2/2     Running   0          127m
+```
+
+## Install Knative Eventing and Kafka Event Source
+- Install [Knative Eventing Core >= 0.18](https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-eventing-component)
+- Install [Kafka Event Source](https://github.com/knative-sandbox/eventing-kafka/releases).
+- Install `InferenceService` addressable cluster role
+
+```
+VERSION=v0.23.0
+kubectl apply --selector knative.dev/crd-install=true --filename https://github.com/knative/eventing/releases/download/$VERSION/eventing-crds.yaml
+kubectl apply -f https://github.com/knative/eventing/releases/download/$VERSION/eventing-core.yaml
+kubectl apply --filename https://github.com/knative/eventing/releases/download/$VERSION/eventing.yaml
+kubectl apply -f https://storage.googleapis.com/knative-releases/eventing-contrib/latest/kafka-source.yaml
+```
+
+```bash
+kubectl apply -f addressable-resolver.yaml
+```
+
+### KAFKA
+```
+kubectl apply -f kafka-client.yaml
+kubectl exec -it kafka-client -- /bin/bash
+kafka-topics --zookeeper my-kafka-cp-zookeeper-headless:2181  --list
+kafka-topics --zookeeper my-kafka-cp-zookeeper-headless:2181 --topic realtime --create --partitions 1 --replication-factor 1 --if-not-exists
+kafka-console-consumer --bootstrap-server my-kafka-cp-kafka-headless:9092 --topic realtime --from-beginning --timeout-ms 4000 --max-messages 5 
+
+```
+
+## Deploy Minio
+- If you do not have Minio setup in your cluster, you can run following command to install Minio test instance.
+```bash
+cd pipeline/MINIO
+kubectl apply -f minio.yaml
+```
+
+- Install Minio client [mc](https://docs.min.io/docs/minio-client-complete-guide)
+
+### MINIO
+
+```
+kubectl port-forward svc/minio-service -n default 9000:9000
+
+mc config host add myminio http://127.0.0.1:9000 minio minio123
+
+mc mb myminio/rawimage
+mc mb myminio/imageprediction
+
+- Setup event notification to publish events to kafka.
+mc admin config set myminio notify_kafka:1 tls_skip_verify="off"  queue_dir="" queue_limit="0" sasl="off" sasl_password="" sasl_username="" tls_client_auth="0" tls="off" client_tls_cert="" client_tls_key="" brokers="my-kafka-cp-kafka-headless:9092" topic="realtime" version=""
+
+# Restart minio
+mc admin service restart myminio
+
+# Setup event notification when putting images to the bucket
+mc event add myminio/rawimage arn:minio:sqs:us-east-1:1:kafka -p --event put --suffix .jpg
+```
+
+## Create the InferenceService
+![](serving.png)
+Specify the built image on `Transformer` spec and apply the inference service CRD.
+```bash
+cd pipeline/SERVING
+kubectl apply -f minio_serving.yaml 
+```
+
```

Comment:
Do we need this step if we create `InferenceService` in our DAG Pipeline ?

Reply:
I think we can remove that line. But we are doing KF serving

Reply:
I guess, you deploy `InferenceService` in you Notebook, so we can remove it from README.

Reply:
Ok sure i will do that